cmd/osmosisd/cmd/root.go

```diff
@@ -401,6 +417,52 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 	return rootCmd, encodingConfig
 }
 
+// overwrites config.toml values if it exists, otherwise it writes the default config.toml
+func overwriteConfigTomlValues(serverCtx *server.Context) (*tmcfg.Config, error) {
+	// Get paths to config.toml and config parent directory
+	rootDir := serverCtx.Viper.GetString(flags.FlagHome)
+	configParentDirPath := filepath.Join(rootDir, "config")
+	configFilePath := filepath.Join(configParentDirPath, "config.toml")
+
+	// Initialize default config
+	tmcConfig := tmcfg.DefaultConfig()
+
+	_, err := os.Stat(configFilePath)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			return nil, fmt.Errorf("failed to read in %s: %w", configFilePath, err)
+		}
+
+		// If does not exist, write default config.toml to update
+
+		// We modify the default config.toml to have faster block times
+		// It will be written by server.InterceptConfigsPreRunHandler
+		tmcConfig.Consensus.TimeoutCommit = 4 * time.Second
```

Comment:
Lets add an if statement, to only override if its 5 seconds. (Someone should be able to manually override still)